chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts

```diff
@@ -29,7 +31,7 @@ export class MockCustomerService {
     return firstName[Math.floor(Math.random() * firstName.length)];
```

Comment:
Shouldn't you be returning the customer object here.

Reply:
The first name is returning from the generateFirstName method.